README.md

```diff
@@ -1 +1,71 @@
-# leaderboard-app
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# leaderboard-app
+
+> This is a leaderboard app that displays leader scores and adds them as well.
+
+![screenshot](./app-screenshot1.png)
```

Comment:
- [ ] Kindly put a valid screenshot here to make your README look more professional